### DISCUSSION TITLE
Error building Component Pinecone

### DISCUSSION THREAD

**Discussion by devarthurguilherme** [langflow-ai/langflow#3564] [Q&A]
Hello guys, I can't to create Database to store my embeddings using Pinecone.

![Captura de tela 2024-08-27 053744](https://github.com/user-attachments/assets/970b0b88-b674-49eb-85f5-b1c37f6e4601)
![Captura de tela 2024-08-27 053822](https://github.com/user-attachments/assets/234ef532-bebb-47f8-9369-dc47959bcce6)

**Comment by ubiqtechai** [comment]
Any solution ??? Facing same issue
